docs/core_docs/docs/integrations/chat/novita.ipynb

```diff
@@ -0,0 +1,207 @@
+    "\n",
+    "<Npm2Yarn>\n",
+    "  @langchain/community @langchain/core\n",
+    "</Npm2Yarn>\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Instantiation\n",
+    "\n",
+    "Now we can instantiate our model object and generate chat completions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "javascript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import { ChatNovitaAI } from \"../novita.js\";\n",
+    "\n",
+    "const llm = new ChatNovitaAI({\n",
+    "    model: \"gryphe/mythomax-l2-13b\",\n",
+    "    temperature: 0,\n",
+    "    // other params...\n",
+    "})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Invocation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "javascript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "const aiMsg = await llm.invoke([\n",
+    "    [\n",
+    "        \"system\",\n",
+    "        \"You are a helpful assistant that translates English to French. Translate the user sentence.\",\n",
+    "    ],\n",
+    "    [\"human\", \"I love programming.\"],\n",
+    "])\n",
+    "aiMsg"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "javascript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "console.log(aiMsg.content)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Chaining\n",
+    "\n",
+    "We can [chain](../../how_to/sequence.ipynb) our model with a prompt template like so:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "javascript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import { ChatPromptTemplate } from \"@langchain/core/prompts\"\n",
+    "\n",
+    "const prompt = ChatPromptTemplate.fromMessages(\n",
+    "    [\n",
+    "        [\n",
+    "            \"system\",\n",
+    "            \"You are a helpful assistant that translates {input_language} to {output_language}.\",\n",
+    "        ],\n",
+    "        [\"human\", \"{input}\"],\n",
+    "    ]\n",
+    ")\n",
+    "\n",
+    "const chain = prompt.pipe(llm);\n",
+    "await chain.invoke(\n",
+    "    {\n",
+    "        input_language: \"English\",\n",
+    "        output_language: \"German\",\n",
+    "        input: \"I love programming.\",\n",
+    "    }\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## API reference\n",
+    "\n",
+    "For detailed documentation of Novita AI LLM APIs, head to [Novita AI API reference](https://novita.ai/docs/model-api/reference/llm/llm.html?utm_source=github_langchain&utm_medium=github_readme&utm_campaign=link)\n"
```

Comment:
We prefer having this link to the LangChain API refs, but this is ok too

Reply:
I'm truly sorry but I don't have more time to write a Novita API ref for LangChain 😭.

Reply:
Oh, no this will be autogenerated don't worry.